### PR TITLE
Make tagging work with new tag input format for ec2_asg

### PR DESCRIPTION
**Pull request by feanil**
It looks like the format of this module changed in one of the upgrades on is not compatible with what we're doing right now.  This resolves it.

Module docs: http://docs.ansible.com/ansible/ec2_asg_module.html

### CODE REVIEW

**Comment by feanil** [comment]
@edx/devops

File: playbooks/edx-east/edx_service.yml [hunk 1]
```diff
@@ -163,19 +163,27 @@
...
+
+    - debug: msg="Instance Tags:{{ listdict_asg_instance_tags }}"
+
     - name: Manage ASG
       ec2_asg:
         profile: "{{ profile }}"
         region: "{{ aws_region }}"
         name: "{{ asg_name }}"
         launch_config_name: "{{ service_config.name }}"
-        load_balancers: "{{ elb_name }}"
```
Comment (maxrothman) [inline playbooks/edx-east/edx_service.yml:172]: Why'd you remove this line?

**Comment by e0d** [comment]
:+1:  we need to write tests for this at some point, can you add a ticket.

**Comment by maxrothman** [comment]
The documentation gives this example:

```
 tags:
    - environment: production
      propagate_at_launch: no
```

As I understand it, you're producing this format:

```
tags:
  - environment: production
  - propegate_at_launch: no
```

Is the documentation incorrect?

File: playbooks/edx-east/edx_service.yml (hunk 1 above)
Comment (feanil) [inline playbooks/edx-east/edx_service.yml:172]: It's in there twice.

Comment (maxrothman) [inline playbooks/edx-east/edx_service.yml:172]: :+1: I see.

**Comment by feanil** [comment]
Ticket created: https://openedx.atlassian.net/browse/OPS-1117

@maxrothman The input I'm taking is not currently in the format you are using.

It's in the format documented in example of when you would use this functien.

**Comment by maxrothman** [comment]
@feanil I'm not sure I got that. Can you explain further?

**Comment by feanil** [comment]
Explained offline.

**Pull request merged**